boilerplate.js

```diff
@@ -227,6 +250,11 @@ async function install(context) {
           "  }",
       },
     )
+
+    ignite.patchInFile(`${process.cwd()}/package.json`, {
+      replace: `"postinstall": "solidarity",`,
+      insert: `"postinstall": "solidarity && jetify && (cd ios; pod install)",`,
+    })
```

Comment:
Just for future reference, `ignite.patchInFile` is going to be deprecated (although probably not removed) and I recommend using `patching.patch` or `patching.replace`. https://github.com/infinitered/gluegun/blob/master/docs/toolbox-patching.md#replace